pkg/controllers/replica-controller/handler.go

```diff
@@ -257,6 +262,24 @@ func (c *CStorVolumeReplicaController) cVREventHandler(
 		if isCVRCreateStatus(cvrObj) {
 			return c.cVRAddEventHandler(cvrObj, fullVolName)
 		}
+
+		if cvrObj.Annotations[restoreCompletedAnnotation] == "true" {
+			targetIp, err := volumereplica.GetTargetIp(fullVolName)
+			if err != nil {
+				return "", err
```

Comment:
Currently, the [caller](https://github.com/openebs/cstor-operators/blob/f9e3ad47b6e0b4e86cbe1e1369f5384ee706df8e/pkg/controllers/replica-controller/handler.go#L116) of `cVREventHandler` will not log any errors is the status is returned as empty(`""`)[If possible good to add error log in caller code -- [here](https://github.com/openebs/cstor-operators/blob/f9e3ad47b6e0b4e86cbe1e1369f5384ee706df8e/pkg/controllers/replica-controller/handler.go#L119)].

I know this is limitation exists with the current code(replica-controller). I always have in mind to refactor volume-replica controller code with the latest learnings.

Reply:
I added logging, PTAL. thanks